ElectronApp/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import App from '../renderer/App';
+
+test('full app rendering/navigating', () => {
```

Comment:
It's best to separate out tests when possible. I know that it might feel a bit contrived since we're just starting out, but imagine if all you saw from your test results was "full app rendering/navigating failed." Imagine we were much further along in development - would you know which part of the app was failing?

Consider moving some of this logic to create a separate test for the router, or perhaps even for each link on the router.